ci/jenkins/test.sh

```diff
@@ -35,7 +35,7 @@ IMAGE_PULL_POLICY="Always"
 PROXY_ALL=false
 DEFAULT_IP_MODE="ipv4"
 IP_MODE=""
-K8S_VERSION="1.28.2-00"
+K8S_VERSION="1.28.9-2.1"
```

Comment:
Kubernete 1.30 has been released, can we upgrade the testbed to the latest one?

Reply:
Fixed.